bokun.yaml

```diff
@@ -5688,56 +5692,73 @@ definitions:
       rateId:
         format: int64
         type: integer
+        description: 'Specifies the rate ID.'
       title:
         type: string
+        description: 'Specifies the title of the rate.'
       price:
         $ref: '#/definitions/PriceDto'
+        description: 'Specifies the rate price if it is set per booking. Field omitted if the price is set per person.'
       passengers:
         items:
           $ref: '#/definitions/CategoryPriceDto'
         type: array
+        description: 'The list of pricing categories and related prices per person. This field is omitted if the price is set per booking.'
       pickupPrice:
         $ref: '#/definitions/PriceDto'
+        description: 'Specifies the price for the pick up service on this rate.'
```

Comment:
This is not a complete description. I suggest simply copying over documentation from https://github.com/Bokun/bokun_platform/blob/master/bokun_api/src/main/java/is/bokun/dtos/pricing/ActivityRatePricesDto.java#L28 (also for other fields), why reinvent the wheel?

Reply:
Good point!